### PR TITLE
pacman: add more information link

### PR DESCRIPTION
**Pull request by navarroaxel**
- [ ] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.fr/linux/pacman.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # pacman
 
-> Outil de gestion de paquets sur Arch Linux.
```
Comment (bl-ue) [inline pages.fr/linux/pacman.md:3]: What changed here?

Comment (bl-ue) [reply pages.fr/linux/pacman.md:3]: Oh just a space. 😄

**Review by bl-ue — APPROVED** [review]

**Comment by bl-ue** [comment]
@navarroaxel how about `കൂടുതൽ വിവരങ്ങൾക്ക്: <...>.` for the Malayalam translation ([`pages.ml/linux/pacman.md`](https://github.com/tldr-pages/tldr/blob/master/pages.ml/linux/pacman.md))?

@avronr @govind-shenoy does that look right?

**Review by schneiderl — APPROVED** [review]

File: pages.fr/linux/pacman.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # pacman
 
-> Outil de gestion de paquets sur Arch Linux. 
+> Outil de gestion de paquets sur Arch Linux.
+> Plus d'informations : <https://man.archlinux.org/man/pacman.8.en>.
```
Comment (sbrl) [inline pages.fr/linux/pacman.md:4]: Slightly shortening this to remove the language code works too. Maybe then it'll display in the user's native language?
```suggestion
> Plus d'informations : <https://man.archlinux.org/man/pacman.8>.
```

Same goes for the other URLs.

Comment (bl-ue) [reply pages.fr/linux/pacman.md:4]: @sbrl links like https://man.archlinux.org/man/pacman.8.fr redirect to the `en` version 🤷

Comment (bl-ue) [reply pages.fr/linux/pacman.md:4]: But removing the code is a good idea still, to be as language agnostic as possible.

File: pages.zh/linux/pacman.md [hunk 3]
```diff
@@ -1,6 +1,7 @@
 # pacman
 
 > Arch Linux 的软件包管理器工具。
+> 更多信息: <https://man.archlinux.org/man/pacman.8.en>.
```
Comment (navarroaxel) [inline pages.zh/linux/pacman.md:4]:
```suggestion
> 更多信息: <https://man.archlinux.org/man/pacman.8>.
```

File: pages/linux/pacman.md [hunk 4]
```diff
@@ -1,6 +1,7 @@
 # pacman
 
 > Arch Linux package manager utility.
+> More information: <https://man.archlinux.org/man/pacman.8.en>.
```
Comment (navarroaxel) [inline pages/linux/pacman.md:4]:
```suggestion
> More information: <https://man.archlinux.org/man/pacman.8>.
```

**Review by sbrl — APPROVED** [review]

**Review by owenvoke — APPROVED** [review]

**Pull request merged**